README.md

```diff
@@ -102,7 +102,7 @@ crewAI supports integration with local models, thorugh tools such as [Ollama](ht
 
 ```python
 from langchain.llms import Ollama
-ollama_openhermes = Ollama(model="agent")
+ollama_openhermes = Ollama(model="openhermes")
```

Comment:
can we make it `openhermes2.5-mistral`? it's a better and still pretty small model everyone should be able to run

Reply:
Thanks! No preference on my end, was just trying to match the name of the variable